lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -912,6 +913,10 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
             self.credential, self.subscription_id, self.cloud
         )
 
+        self._msi_client = get_managed_service_identity_client(
```

Comment:
Use a method in common.py, instead of initializing every time.

Reply:
Don't save the msi in platform. Call a method in common, when it's needed.

Reply:
Done